22-ajax-lab/src/01-swapi.jsx

```diff
@@ -1,28 +1,145 @@
-/**
- * Lab 01:
- *
- * Write two components for showing data from swapi.co:
- *
- * Person component should show information about a specific character.
- * Film component should show information about a specific film.
- *
- * Both Person and Film show a list of the corresponding film/person,
- * allowing a user to expand from a Person to a Film she played in,
- * to another Person from that film and so on.
- */
-
-import React from 'react';
-
-const Person = React.createClass({
-});
-
-const Film = React.createClass({
-});
-
-export default React.createClass({
-  render() {
-    return <div>
-      <Person id={1} />
-    </div>
-  }
-});
+/**
+ * Lab 01:
+ *
+ * Write two components for showing data from swapi.co:
+ *
+ * Person component should show information about a specific character.
+ * Film component should show information about a specific film.
+ *
+ * Both Person and Film show a list of the corresponding film/person,
+ * allowing a user to expand from a Person to a Film she played in,
+ * to another Person from that film and so on.
+ */
+import React from "react";
+
+const PEOPLE_PREFIX = "http://swapi.co/api/people/";

```

Comment:
I suggest to wrapping base api in a function and call it let's say
getApiFor in the following manner:

```
function getApiFor(apiEndpoint, baseUrl = "http://swapi.co/api"){
   return `${baseUrl}/${apiEndpoint}`
}
```

So the consts will look as following:
```
const PEOPLE_PREFIX = getApiFor('people');
const FILMS_PREFIX = getApiFor('films');
```